### DISCUSSION TITLE
Welcome to FastKafka Discussions!

### DISCUSSION THREAD

**Discussion by davorrunje** [airtai/fastkafka#2] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by sraeuchle** [comment]
First one!

I'm testing your library if it works for my use-case (minimal code to add AI on my Kafka enrichment data streams)

I wonder how to start a FastKafka Application without a cli (fastkafka run ...). for example to put my application into a docker or start nicely on PyCharm using python commands.

any hints will be welcome :)

something like:

```python

# application.py

kafka_app = FastKafka(
    title=application_name,
    kafka_brokers=brokers
)

# Options maybe?!
kafka_app.run(..)
FastkKafka.run(kafka_app)

```

**Reply by hajdicenan** [reply]
Hi! Take a look at this guide and let us know if it helps: https://fastkafka.airt.ai/docs/guides/Guide_30_Using_docker_to_deploy_fastkafka. 
Also, if easier, you can find us on Discord and ping us there: https://bit.ly/fastkafka-discord :-)

**Reply by sternakt** [reply]
Hi!

Also, if you want to run the app in your .py file (something like the example you provided), you can use the app as an async context manager.

```python 
kafka_app = FastKafka(
    title=application_name,
    kafka_brokers=brokers
)

async with kafka_app:
    # your code here
```
